YAML/sublime-syntax.sublime-syntax

```diff
@@ -453,9 +453,9 @@ contexts:
   variable_call:
     - match: '(\{\{)([a-zA-Z_0-9]*)(\}\})'
       captures:
-        1: punctuation.definition.parameters.begin.syntax
+        1: punctuation.definition.arguments.begin.syntax
```

Comment:
for my use, I added the line: `0: variable.parameter` so that variables would be highlighted by the color scheme and thus be easier to spot :)